api/endpoints.go

```diff
@@ -278,6 +307,7 @@ func (eh *endpointHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Write response.
 	w.Header().Set("Content-Type", apiEndpoint.MimeType+"; charset=utf-8")
 	w.Header().Set("Content-Length", strconv.Itoa(len(responseData)))
+	w.Header().Set("X-Content-Type-Options", "nosniff")
 	w.WriteHeader(http.StatusOK)
```

Comment:
Well it doesn't harm but there's also no real security improvement by using `nosniff` here as the portmaster cannot be tricked into serving otherwise user/attacker generated content anyway.

Reply:
I just followed how `http.Error` does it. And I didn't want to reason with the stdlib. ;)